################################################################################
SHORT DESCRIPTION:
################################################################################
Enable building xen tools against external blktap3 library 

################################################################################
LONG DESCRIPTION:
################################################################################
By default, xen uses blktap from inside the xen tree. Modified to link with 
blktap3 from the system libraries.

################################################################################
CHANGELOG
################################################################################
Authors:
Mahantesh Salimath <salimathm@ainfosec.com>

################################################################################
PATCHES
################################################################################
Index: xen-4.9.0/tools/Rules.mk
===================================================================
--- xen-4.9.0.orig/tools/Rules.mk
+++ xen-4.9.0/tools/Rules.mk
@@ -169,7 +169,7 @@ ifeq ($(LIBXL_BLKTAP),y)
 CFLAGS_libblktapctl = -I$(XEN_BLKTAP2)/control -I$(XEN_BLKTAP2)/include $(CFLAGS_xeninclude)
 SHDEPS_libblktapctl =
 LDLIBS_libblktapctl ?= $(SHDEPS_libblktapctl) $(XEN_BLKTAP2)/control/libblktapctl$(libextension)
-SHLIB_libblktapctl  = $(SHDEPS_libblktapctl) -Wl,-rpath-link=$(XEN_BLKTAP2)/control
+SHLIB_libblktapctl  = $(SHDEPS_libblktapctl) -lblktapctl
 else
 CFLAGS_libblktapctl =
 SHDEPS_libblktapctl =
